Description: Look for .so plugins in /usr/lib/opencpn-plugins/
 (lintian E: arch-dependent-file-in-usr-share)
Author: Hamish B <hamish_b yahoo com>
Index: opencpn/src/pluginmanager.cpp
===================================================================
--- opencpn.orig/src/pluginmanager.cpp	2011-06-15 07:31:01.063910240 +1200
+++ opencpn/src/pluginmanager.cpp	2011-06-15 08:31:00.678910033 +1200
@@ -126,7 +126,8 @@
 
 bool PlugInManager::LoadAllPlugIns(wxString &shared_data_prefix)
 {
-      m_plugin_location = shared_data_prefix + _T("plugins");
+      //m_plugin_location = shared_data_prefix + _T("plugins");
+      m_plugin_location = _T("/usr/lib/opencpn-plugins");
       wxDir pi_dir(m_plugin_location);
 
       wxString msg(_("PlugInManager searching for PlugIns in location "));
